### PR TITLE
add RCs to beta channel for testing purpose

### PR DESCRIPTION
**Pull request by davitol**
@VicDeo @PVince81 I will test this updates via beta channel using the RCs.

updater PR for final tarballs will come later

### CODE REVIEW

File: update-server/tests/integration/features/update.beta.feature [hunk 1]
```diff
@@ -42,7 +42,7 @@ Feature: Testing the update scenario of releases on the beta channel
     And The received version is "9.0.10"
     When The request is sent
     Then The response is non-empty
-    And URL to download is "https://download.owncloud.org/community/owncloud-9.1.6.zip"
+    And URL to download is "https://download.owncloud.org/community/owncloud-9.0.11RC2.zip"
```
Comment (PVince81) [inline update-server/tests/integration/features/update.beta.feature]: 9.1.7RC2, we serve the next major update ?

Comment (davitol) [reply update-server/tests/integration/features/update.beta.feature]: If it is the latest minor version, should be don't do it?

File: update-server/config/config.php [hunk 2]
```diff
@@ -227,19 +227,19 @@
...
 		// START: Due do a bug in the updater we need to enforce the update order
 		// see https://github.com/owncloud/administration-internal/issues/19
 		'9.0' => [
-			'latest' => '9.0.11',
-			'downloadUrl' => 'https://download.owncloud.org/community/testing/owncloud-9.0.11RC2.zip',
-			'web' => 'https://doc.owncloud.org/server/9.0/admin_manual/maintenance/upgrade.html',
+			'latest' => '9.1.7',
+			'downloadUrl' => 'https://download.owncloud.org/community/testing/owncloud-9.1.7RC2.zip',
+			'web' => 'https://doc.owncloud.org/server/9.1/admin_manual/maintenance/upgrade.html',
 		],
```
Comment (VicDeo) [inline update-server/config/config.php]: @davitol it's not correct. any 9.0.x should get to 9.0.11 before the latest 9.1.x is allowed

**Comment by VicDeo** [comment]
@PVince81 @davitol history squashed, 9.0/9.1 updated to release packages.

**Comment by davitol** [comment]
👍

**Comment by PVince81** [comment]
Let's deploy next week, shortly before announcement

**Pull request merged**